tests/TestCase/Network/Http/Adapter/StreamTest.php

```diff
@@ -13,8 +13,8 @@
  */
 namespace Cake\Test\TestCase\Network\Http\Adapter;
 
-use Cake\Network\Http\Adapter\Stream;
-use Cake\Network\Http\Request;
+use Cake\Http\Client\Adapter\Stream;
+use Cake\Http\Client\Request;
```

Comment:
Yeah for Projekts they should be Red probably. But Core itself should at some point not internally use deprecations.